README.md

```diff
@@ -1 +1,71 @@
-# Ruby-capstone
\ No newline at end of file
+# Catalog of My Things
+ 
+> Imagine that you are in a catalog of a book and a game store, and you need a tool to record what books. The app that you will create will allow you to:
+
+- Add books.
+- Add games.
+- See the books.
+- See the games.
+- See the labels.
+- See the authors.
+
+And all of this will be built in a beautiful and well-organized way!
+
+## Built With
+
+- Ruby
+- Linters: [Rubocop](https://rubocop.org/)
+- Visual Studio Code
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and Ruby.
+
+For Ruby installation.
+```
+$ ruby --version && irb
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Clone
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/renercrows/Ruby-capstone.git
+```
+
+## Authors
+
+👤 **Maximiliano Ungredda**
+- GitHub: @renercrows
+- Twitter: @renercrows
+- LinkedIn: (www.linkedin.com/in/maximiliano-ungredda)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/renercrows/OOP-School-Library/tree/project1/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Kanban template
+
+## 📝 License
+
+This project is MIT licensed.
```

Comment:
- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯